locale/en_US/locale.xml

```diff
@@ -26,6 +26,7 @@
 	<message key="plugins.generic.browseBySection.browseByPerPageLabel">Items Per Page</message>
 	<message key="plugins.generic.browseBySection.browseByPerPageDescription">How many items should be displayed per page on your section archive.</message>
 	<message key="plugins.generic.browseBySection.browseByDescriptionLabel">Section Archive Description</message>
+	<message key="plugins.generic.browseBySection.browseByOrder">Options to sort by</message>
```

Comment:
To avoid adding a new locale key, we may want to consider:
* `grid.action.order`
* `manager.statistics.reports.orderBy`
* `grid.action.sort`